src/Configuration/Config.Abstractions/ref/Microsoft.Extensions.Configuration.Abstractions.netstandard2.0.cs

```diff
@@ -44,6 +44,7 @@ public partial interface IConfigurationProvider
         void Load();
         void Set(string key, string value);
         bool TryGet(string key, out string value);
+        void Use(System.Collections.Generic.IDictionary<string, string> data);
```

Comment:
This is a breaking change... We shouldn't do that with extensions unless warranted. Why does this pass the bar?

Reply:
OK so I spent some time looking at this and this breaks the IConfiguratonProvider abstraction generally. Pushing data into the IConfigurationProvider doesn't make sense as it's supposed to be aware of where it got the data from. It cannot be fed data.